website/src/components/LanguageSelector.module.scss

```diff
@@ -0,0 +1,11 @@
+$size: 24px;
```

Comment:
You can use native [css variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) instead of using sass.